### PR TITLE
Added configuration options multimailhook.smtpServerTimeout (in seconds)...

### PR DESCRIPTION
**Pull request by mikkoi**
... and multimailhook.smtpServerDebug (SMTP debug) and their use.

To help finding faults with cranky SMTP servers.

### CODE REVIEW

**Comment by moy** [comment]
@mikkol: Thanks for the patch. But would you please add a `Signed-off-by` line to the commit, indicating your acceptance of the license [in the manner of the Git project](https://github.com/git/git/blob/9532ead9875b7b92ff2007f9de61af5874e8839a/Documentation/SubmittingPatches#L234)? Feel free to force-push to your branch with the fixed commit. Thanks!

**Comment by mikkol** [comment]
Hi! Looks like it's @mikkoi who's behind the commit, not me.

**Comment by mikkoi** [comment]
Yep, it me. @mikkoi.
Shall be done.

**Comment by moy** [comment]
Sorry for the confusion with usernames!

It seems PR #63 does about the same thing (but yours has doc ;-)). Can you check if this PR does all what #63 does?

Thanks,

File: git-multimail/README [hunk 1]
```diff
@@ -268,6 +268,25 @@ multimailhook.mailer
...
+           True/False. Set True to see all debug messages to/from
+           SMTP server.
+
+       multimailhook.smtpServerSecurity
+
+           Set the security type. Allowed values: none, ssl.
+           Default=none.
+
+       multimailhook.smtpServerUsername
+       multimailhook.smtpServerPassword
```
Comment (moy) [inline git-multimail/README]: I would have started with username/password/security, and put debug and timeout after.

**Comment by mikkoi** [comment]
Hi, We can trash this pull-request. I created a new one from a different branch (smtp-server-conf) as it should be. Commit message exceeded 50 chars.

**Comment by moy** [comment]
Superseeded by #85.

**Pull request closed without merging**